tezos/new_context/bin/context_action_file_replayer.rs

```diff
@@ -340,6 +341,7 @@ fn resolve_context_kv_store(
     match context_kv_store_configuration {
         ContextKvStoreConfiguration::Sled { path } => ("sled".to_string(), Some(path.clone())),
         ContextKvStoreConfiguration::InMem => ("inmem".to_string(), None),
+        ContextKvStoreConfiguration::InMemGC => ("inmem-gc".to_string(), None),
```

Comment:
@sebastiencs @tizoc 
this is not a best way, I did not want to mix K-V with Garbage collection algorithm,
if it is just temporary fix, ok, but final solution should be:
- I choose k-v store on startup
- I choose GC algorithm on startup

Reply:
@bkontur I know, and it is very likely that we will end having a few more settings for the storage, not just the GC.